quarkchain/cluster/cluster_config.py

```diff
@@ -415,7 +419,10 @@ def __create_from_args_internal():
                 slave_config.PORT = args.port_start + i
                 slave_config.ID = "S{}".format(i)
                 slave_config.CHAIN_MASK_LIST = [ChainMask(i | args.num_slaves)]
-
+                if i < 2:
```

Comment:
what does this mean?

Reply:
since we have 4 slaves, just for testing, the first 2 slaves are still QKC, the rest are LIBRA

Reply:
but you do know slave number is configurable, right? it can be 8, 16, or whatever number

Reply:
do you mean I can set random id for libra slave?

Reply:
I mean your logic in line 422 will break if in production we have different number of slaves in config. 

I would suggest in command-line flags, don't allow setting types for slave servers

Reply:
done